test/Electrical/analog.jl

```diff
@@ -276,7 +276,7 @@ _damped_sine_wave(x, f, A, st, ϕ, d) = exp((st - x) * d) * A * sin(2 * π * f *
 
     for i in 1:lastindex(sources)
         source = sources[i]
-        @info "Testing Voltage with $(source.name) source"
+        @info "Testing Voltage with $(source.get_name) source"
```

Comment:
This works:

```suggestion
        @info "Testing Voltage with $(ModelingToolkit.getname(source)) source"
```

Better still - a `using ModelingToolkit: getname` at the top and `$(getname(source))` here.

Reply:
This just makes the change in a string naming the test. Not the actual code.